back-office/customer-service/customer-service/dashboard.md

```diff
@@ -0,0 +1,362 @@
+
+It's possible to sort by employee asc or desc and search by employee's lastname.
+
+8) Messages
+
+The customer's message is displayed and truncated if it contains more than 40 chars.
+
+It's possible to sort by message asc or desc and search by messages
+
+9) Private
+
+An icon is displayed according to the privacy of the message: a cross if the message is not private (sent to the customer) and a tick if the message is private (not sent to the customer)
+
+It's possible to sort by privacy asc or desc and search by selecting the privacy through the drop-down, where 2 options are listed: "yes" and "no".
+
+10) Last message
+
+The date and hour of the last customer's message is displayed.
+
+It's possible to sort by date asc or desc and search by selecting a range of date through two datepicker: "from" and "to"
+
+11) Shop
+
+If the multistore is enabled and several shops have been configured, then this column is displayed.
+The name of the shop where the message was sent is displayed.
+
+It's not possible to sort or perform a search for this column.
+
+12) Actions
+
+There are two actions:
+
+- View, the main one:
+
+After clicking on it, the thread is opened in the same tab.
+This action can also be performed by clicking on a line.
+
+- Delete, the second one:
+
+After clicking on it, a modal is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?"
+
+The action can be canceled by clicking on the cross or on the cancel button.
+
+After clicking on "Delete" button, the message is deleted and a successful message is displayed "Successful deletion."
+
+#### Search
+
+Below, the same behavior as others listing:
+
+By default, the search button is disabled. It becomes enabled when something is filled in a search field.
+
+After clicking on "Search", the search is applied and the messages matching with the search are displayed
+
+If no results are found, then a message "No records found" is displayed.
+
+After clicking on "Search", this button is disabled and a "Reset" button is displayed.
+
+After clicking on "Reset", the search is removed and all messages are displayed.
+
+#### Pagination
+
+The pagination of the list starts when there are more than 20 messages.
+
+The merchant can select the number of messages displayed through a drop-down: 20, 50, 300, 1000
+
+When there are more results than the number selected, the merchant can navigate through pages.
+
+If a sort and/or a search have been done, results should be kept when moving on to the next page.
+
+## CONTACT OPTIONS
+
+The user can edit 2 options, which apply to all contacts:
+
+1) **Allow file uploading**: This toggle allows the merchant to choose if he/she wants to allow his/her visitors to upload files through the contact form.
+
+It's enabled by default.
+
+When enabled, an optional field "Attachment" is displayed in the front-office's contact form and a "choose file" CTA allows to select a file.
```

Comment:
```suggestion
When enabled, an optional field "Attachment" is displayed in the front-office's contact form and a "choose file" CTA allows to select a file.

Only the following extension are accepted: pdf, txt, png, jpg, gif, zip.
Otherwise, an error notfication is displayed: "Bad file extension".
```